docs/api.md

```diff
@@ -5,9 +5,21 @@ GET /institutions
 returns a paginated list of institutions
 
 parameters:
-page:string, optional, default='1'
+page:int, optional, default='1'

```

Comment:
Shouldn't that be changed also in swagger config file? idk just guessing

Reply:
We should but developong swagger is kinda on hold because the single markdown file does the job fine.